utils/ledger/reportLedgerConfigError/index.js

```diff
@@ -7,7 +7,8 @@ export const reportLedgerConfigError = ({
   inUseByAnotherApp,
   otherError
 }) => {
-  if (connectedFailure) return 'Is your Ledger device plugged in?'
+  if (connectedFailure)
+    return 'Is your Ledger device plugged in? Try again or refresh the page to return home'
```

Comment:
I'm going to remove this for now. This error card is not the only place where this error message will appear. If we really find users getting that aggravated over this, we can make copy improvements or add the cancel button.